python/ray/serve/_private/deployment_scheduler.py

```diff
@@ -38,6 +40,11 @@ class SpreadDeploymentSchedulingPolicy:
 
 @total_ordering
 class Resources(dict):
+    # Custom resource priority from environment variable
+    CUSTOM_PRIORITY = [
+        r for r in RAY_SERVE_HIGH_PRIORITY_CUSTOM_RESOURCES.split(",") if r
+    ]
```

Comment:
nit: I would prefer the `.split` logic to be `constants.py` purely for consistency with other constants in the file.

Reply:
+1